### DISCUSSION TITLE
Pittboss RTD probe

### DISCUSSION THREAD

**Discussion by scoob8000** [nebhead/PiFire#75] [Q&A]
I seem to recall seeing someone used their stock Pitboss RTD probe in their build but I forget where I saw that.

I already ordered the trager part, but if I can re-use my stock one that will make swapping back and forth easier.

Any to anyone that has, can you verify the connector on it is a JST SM plug?  

It sure looks like it:
Connectors:
https://www.amazon.com/Connector-VANDESAIL-Female-Adapter-Electrical/dp/B076HLQ4FX

Pitboss probe:
https://www.amazon.com/Replacement-Temperature-Sensor-Suitable-PB1000XL-025-R00

**Comment by nebhead** [comment]
I don't know the answer to this one, but the probe looks very similar to the Traeger part.  Also it's part name suggests that it might be an RTD1000 which would be identical.  That said, you can potentially just connect it to PiFire (I think your JST connectors look correct there) and run through the probe tuning.  That would mean you'll need a reference probe/thermometer, starting with a cold grill run it up to at least 350-400F and take measurements at low/medium/high temps with the tuning tool (on the settings page of PiFire).

**Reply by scoob8000** [reply]
Sounds like a plan.  I'm gonna get a set and give it a go.   I hate to buy 20 pairs, but by the time I order one connector from Mouser and pay the shipping it's gonna be about the same.  :)

**Comment by gsherwin74** [comment]
I wired my stock Pitboss probe directly in to the PiFire system and everything works perfectly.

**Reply by scoob8000** [reply]
Did yours have the little SM connectors on them or did you cut the leads to connect to the pi fire?

I'm pretty certain those leads I found are the same, I may try to connectorize them on the pifire to make easier to revert back to the stock controller (just by swapping the plugs over.
